lifecycle/export.go

```diff
@@ -0,0 +1,133 @@
+package lifecycle
+
+import (
+	artUtils "github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	artServices "github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	clientConfig "github.com/jfrog/jfrog-client-go/config"
+	"github.com/jfrog/jfrog-client-go/lifecycle/services"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+type ReleaseBundleExportCommand struct {
+	releaseBundleCmd
+	modifications          services.Modifications
+	downloadConfigurations artUtils.DownloadConfiguration
+	targetPath             string
+}
+
+func (rbe *ReleaseBundleExportCommand) Run() (err error) {
+	if err = validateArtifactoryVersionSupported(rbe.serverDetails); err != nil {
+		return
+	}
+	servicesManager, rbDetails, queryParams, err := rbe.getPrerequisites()
+	if err != nil {
+		return errorutils.CheckErrorf("Failed getting prerequisites for exporting command, error: '%s'", err.Error())
+	}
+	// Start the Export process and wait for completion
+	log.Info("Exporting Release Bundle archive...")
+	exportResponse, err := servicesManager.ExportReleaseBundle(rbDetails, rbe.modifications, queryParams)
+	if err != nil {
+		return errorutils.CheckErrorf("Failed exporting release bundle, error: '%s'", err.Error())
+	}
+	// Download the exported bundle
+	log.Debug("Downloading the exported bundle...")
+	_, failed, err := rbe.downloadReleaseBundle(exportResponse, rbe.downloadConfigurations)
```

Comment:
Might want not to ignore the `downloaded int` to assert the release bundle was truly downloaded.